rest-api/tools/import_organizations.sh

```diff
@@ -37,3 +49,10 @@ fi
 source tools/set_path.sh
 python tools/import_organizations.py --awardee_file data/awardees.csv \
   --organization_file data/organizations.csv --site_file data/sites.csv $DRY_RUN
+
+function finish {
+  rm -rf ${TMP_GEOCODE_DIR}
```

Comment:
I believe that the trap below overrides the one inside auth_setup.sh (which calls cleanup.) So you need to do something like:

function cleanup {
}

at the top of the file... and then call cleanup in finish, so if auth_setup overrides it it still gets called.